inclusion_connect/accounts/tests.py

```diff
@@ -103,6 +104,70 @@ def test_user_creation_terms_are_required(client):
     assert "terms_accepted" in response.context["form"].errors
 
 
+def test_account_activation(client):
+    redirect_url = reverse("oidc_overrides:logout")
+    url = add_url_params(reverse("accounts:activation"), {"next": redirect_url})
+    user = UserFactory.build()
+    assert not get_user(client).is_authenticated
+
+    # If missing params in oidc session
+    response = client.get(url)
+    assert response.status_code == 400
+
+    client_session = client.session
+    client_session[OIDCSessionMixin.OIDC_SESSION_KEY] = {
+        "email": user.email,
+        "firstname": user.first_name,
+        "lastname": user.last_name,
+    }
+    client_session.save()
+    response = client.get(url)
+    assertTemplateUsed(response, "account_activation.html")
+
+    response = client.post(
+        url,
+        data={
+            "email": user.email,
+            "first_name": user.first_name,
+            "last_name": user.last_name,
+            "password1": DEFAULT_PASSWORD,
+            "password2": DEFAULT_PASSWORD,
+            "terms_accepted": True,
+        },
+    )
+    assertRedirects(response, redirect_url, fetch_redirect_response=False)
+    assert get_user(client).is_authenticated
```

Comment:
Évite des erreur si `is_authenticated` est une méthode.
```suggestion
    assert get_user(client).is_authenticated is True
```